src/pages/evm/staking/StakingPageHeader.vue

```diff
@@ -3,87 +3,158 @@ import { computed, onBeforeMount, ref } from 'vue';
 import { useI18n } from 'vue-i18n';
 import { BigNumber, ethers } from 'ethers';
 
-import { useUserStore, useBalancesStore, useChainStore } from 'src/antelope';
-import { prettyPrintCurrency } from 'src/antelope/stores/utils/currency-utils';
+import { useUserStore, useBalancesStore, useChainStore, useRexStore } from 'src/antelope';
+import { convertCurrency, prettyPrintCurrency } from 'src/antelope/stores/utils/currency-utils';
 import { NativeCurrencyAddress } from 'src/antelope/types';
 import EVMChainSettings from 'src/antelope/chains/EVMChainSettings';
 import { WEI_PRECISION, formatWei } from 'src/antelope/stores/utils';
 
 import ScrollableInfoCards from 'components/evm/ScrollableInfoCards.vue';
 
-
+const label = 'current';
 const { t: $t } = useI18n();
 const userStore = useUserStore();
 const balancesStore = useBalancesStore();
 const chainStore = useChainStore();
 const chainSettings = (chainStore.loggedChain.settings as EVMChainSettings);
 // data
-const systemTokenPrice = ref(0); // always equal to wrapped token price
 const loading = ref(true);
 
 
 // computed
 const fiatLocale = computed(() => userStore.fiatLocale);
 const fiatCurrency = computed(() => userStore.fiatCurrency);
-const systemTokenName = computed(() => chainSettings.getSystemToken().name);
-const systemTokenSymbol = computed(() => chainSettings.getSystemToken().symbol);
-const stakedTokenSymbol = computed(() => chainSettings.getStakedSystemToken().symbol);
+const systemToken = chainSettings.getSystemToken();
+const stakedToken = chainSettings.getStakedSystemToken();
 
 const totalFiatValueText = ref('$ 4,324.35');
 
-const stakedFiatValueText = ref('$ 4,000.23');
-const stakedTokenBalanceBn = ref(ethers.utils.parseUnits('34021.230', WEI_PRECISION));
-const unstakingFiatValueText = ref('$ 200.72');
-const unstakingBalanceBn = ref(ethers.utils.parseUnits('1242.0312', WEI_PRECISION));
-const withdrawableFiatValueText = ref('$ 123.02');
-const withdrawableBalanceBn = ref(ethers.utils.parseUnits('245.0225', WEI_PRECISION));
+// First cell: Staked
+const stakedTokenBalanceBn = computed(() => balancesStore.getBalances(label).find(balance => balance.token.symbol === stakedToken.symbol)?.amount);
+const unstakedRatio = computed(() => chainStore.getUnstakedRatio(label));
+const isStakedLoading = computed(() => stakedTokenBalanceBn.value === undefined || unstakedRatio.value.isZero());
+const stakedExpressedInSystemBalanceBn = computed(() => {
+    if (stakedTokenBalanceBn.value && !unstakedRatio.value.isZero()) {
+        const ratioNumber = ethers.utils.formatUnits(unstakedRatio.value, stakedToken.decimals);
+        console.log('ratioNumber', ratioNumber);
```

Comment:
left this log here
I also see the same ratio number then convercurrency calls, maybe we can do a calculateFiatValue function here that can be reused throughout.

Reply:
removed de console.log
The `ratioNumber` calculation is just one line, I don't think we need a function for that.